src/components/app-registration/AppRegistration/AppRegistration.scss

```diff
@@ -377,20 +379,28 @@ fieldset {
   display: none;
 }
 
-@media screen and (min-width: 480px) {
+[data-state*="responsive.mobile"] {
```

Comment:
Question: why do we use data-state for responsiveness instead of media queries? Looks a bit clumsy here..